specification/support/Support.Management/models.tsp

```diff
@@ -0,0 +1,830 @@
+   */
+  additionalEmailAddresses?: string[];
+
+  /**
+   * Phone number. This is required if preferred contact method is phone. It is also required when submitting 'critical' or 'highestcriticalimpact' severity cases.
+   */
+  phoneNumber?: string;
+
+  /**
+   * Time zone of the user. This is the name of the time zone from [Microsoft Time Zone Index Values](https://support.microsoft.com/help/973627/microsoft-time-zone-index-values).
+   */
+  preferredTimeZone?: string;
+
+  /**
+   * Country of the user. This is the ISO 3166-1 alpha-3 code.
+   */
+  country?: string;
+
+  /**
+   * Preferred language of support from Azure. Support languages vary based on the severity you choose for your support ticket. Learn more at [Azure Severity and responsiveness](https://azure.microsoft.com/support/plans/response/). Use the standard language-country code. Valid values are 'en-us' for English, 'zh-hans' for Chinese, 'es-es' for Spanish, 'fr-fr' for French, 'ja-jp' for Japanese, 'ko-kr' for Korean, 'ru-ru' for Russian, 'pt-br' for Portuguese, 'it-it' for Italian, 'zh-tw' for Chinese and 'de-de' for German.
+   */
+  preferredSupportLanguage?: string;
+}
+
+/**
+ * Describes the properties of a communication resource.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "For backward compatibility"
+model CommunicationDetailsProperties {
+  /**
+   * Communication type.
+   */
+  @visibility(Lifecycle.Read)
+  communicationType?: CommunicationType;
+
+  /**
+   * Direction of communication.
+   */
+  @visibility(Lifecycle.Read)
+  communicationDirection?: CommunicationDirection;
+
+  /**
+   * Email address of the sender. This property is required if called by a service principal.
+   */
+  sender?: string;
+
+  /**
+   * Subject of the communication.
+   */
+  subject: string;
+
+  /**
+   * Body of the communication.
+   */
+  body: string;
+
+  /**
+   * Time in UTC (ISO 8601 format) when the communication was created.
+   */
+  @visibility(Lifecycle.Read)
+  // FIXME: (utcDateTime) Please double check that this is the correct type for your scenario.
+  createdDate?: utcDateTime;
+}
+
+/**
+ * Describes the properties of a Chat Transcript Details resource.
+ */
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-provisioning-state" "For backward compatibility"
+model ChatTranscriptDetailsProperties {
+  /**
+   * List of chat transcript communication resources.
+   */
+  @OpenAPI.extension("x-ms-identifiers", #[])
+  messages?: MessageProperties[];
+
+  /**
+   * Time in UTC (ISO 8601 format) when the chat began.
+   */
+  @visibility(Lifecycle.Read)
+  // FIXME: (utcDateTime) Please double check that this is the correct type for your scenario.
+  startTime?: utcDateTime;
+}
+
+/**
+ * Describes the properties of a Message Details resource.
+ */
+model MessageProperties {
+  /**
+   * Content type.
+   */
+  @visibility(Lifecycle.Read)
+  contentType?: string;
```

Comment:
This should be an enum? https://github.com/Azure/azure-rest-api-specs/blob/main/specification/support/resource-manager/Microsoft.Support/stable/2024-04-01/support.json